files/2da/2da_bgee/wspecial.htm

```diff
@@ -24,3 +26,7 @@
   </div>
 </div>
 
+{% capture note %}
+The <code>SPEED</code> column is ignored by the engine &mdash; High Mastery and Grand Mastery do not provide bonuses to <a href="{{ '/file_formats/ie_formats/itm_v1.htm#itmv1_Extended_Header_SpeedFactor' | prepend: relurl }}"><code>speed factor</code></a> as they should.
+{% endcapture %}
+{% include bug.html %}
```

Comment:
let's wait a bit, likely this is broken in the originals as well.

Reply:
> let's wait a bit, likely this is broken in the originals as well.

As usual, I cannot test the originals...

Reply:
> yeah, but it was asked on the forum, so let's see if anyone replies.

Added (as per [Insomniator](https://www.gibberlings3.net/forums/topic/34892-code-code-fixes-aka-stuff-we-cant-fix/?do=findComment&comment=311576))

If you trust this user, then I guess you can merge this PR...